1.4/Defs/TradeValidationDefs.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+	<DynamicTradeInterface.Defs.TradeValidationDef>
+		<defName>DF_InsufficientSilver</defName>
+		<validation>DynamicTradeInterface.UserInterface.TradeValidation.InsufficientSilver:Validate</validation>
```

Comment:
In the other Defs, i postfixed all the method references with "CallbackHandler"

Reply:
I have renamed this field to `validationCallbackHandler` in the latest version of the PR.